### DISCUSSION TITLE
Is playing with Nvidia became stable?

### DISCUSSION THREAD

**Discussion by Vlad55432** [vinegarhq/sober#36] [General]
I just had one day ago glibc pthread crash (even after update), not sure about how about others

**Comment by AppliedPlumbing2** [comment]
my uncle works at nvidia and nintendo and he said yes and but you have to buy a new nvidia graphic card every day for it to be stable


and then he throwed a amd card on the floor and peed on it saying open source sucks!!! we love propritary drivers!!! screw you jerry sanders!!!! and he said these hash tags out loud **#DownWithMesa #x11forLyfe** also the pee was green because he worked at nvidia
![ynkx0bxg8ced1](https://github.com/user-attachments/assets/f03a74b8-0f84-4d99-90b8-3024b4721a05)

**Reply by Vlad55432** [reply]
Dude...

![IMG_20240806_012108_447.jpg](https://github.com/user-attachments/assets/e9f3334a-f73b-4fa0-9b66-6e09974421ed)

**Comment by km9l** [comment]
nvidia users often has more problems than other users but its 'usable'

**Comment by JimBimBum** [comment]
I'm Mostly curious what the differences are in the way it attempts to use vulkan as before on wine there were none of these issues with robox's built in vulkan when changing Fflags to use it. though did make everything kinda shiny before. also curious if it's possible to pass through DX calls if possible since linux has DXVK native. yes games have been crashing left and right but are game specific from what i can tell.

**Reply by km9l** [reply]
> also curious if it's possible to pass through DX calls if possible since linux has DXVK native.

the android build doesnt have directx which is a windows api

**Reply by JimBimBum** [reply]
i know it doesn't but if valve can somehow put dxvk into their linux ports of games, there might be some trace of dx still in the android apk of roblox leftover. i assume they don't just remove it on compile but they can't exactly execute either cause ya know Android.

**Comment by Vlad55432** [comment]
I can update, that for my machine sober became half stable, still noticing segfaults D:

**Comment by Vlad55432** [comment]
Tunis so cooked, that it became fully stable for my system for 2 days! Yay!

**Comment by Duckrinium** [comment]
It's actually funny how I get explicit sync (sort of an NVIDIA issue) crashes but I cannot actively debug it because it's so random and running it through terminal make it even rarer.

**Comment by SystemEdmond** [comment]
Yes, but with X11

**Comment by Bugsbane** [comment]
I'm using it with an old Geforce 2060+Wayland on KDE Neon, and generally it runs fine! There is one regular glitch, where sometimes it seems to flicker between the current frame and back a few frames ago, but even when it does this it's still perfectly playable.

**Reply by Bugsbane** [reply]
Yes, it's a derivative distro of Ubuntu LTS, maintained by the KDE folks with the latest KDE packages:
https://neon.kde.org/

**Reply by Bugsbane** [reply]
There are two versions of Neon and I think you may be thinking of the other one. The User Edition is entirely a stable, end user focused distro. It uses the long term support version of Ubuntu and only the stable release of KDE. It is fully tested before release like other end user distros. This is what I am using.

KDE Neon ***Testing Edition*** though, is a rolling release using nightly compiled versions of all packages. Yes this is used for testing, but it's not what I'm using.

You can see both here:
https://neon.kde.org/download

Also Neon does support Nouveau (maybe Testing Edition doesn't?). I'm using the proprietary Nvidia drivers myself, and I definitely didn't manually install them, beyond picking a package from the package manager.

**Comment by bingchris** [comment]
nvidia+x11 user here, sober works fine for me

**Comment by JimBimBum** [comment]
they still haven't fixed the vram issue. never had this problem on wine, dxvk or even TF2 which now uses vulkan. anyone demanding to use nouveau can leave cause no one will use an inferior and worse performing driver just to play a game.

**Comment by Nekoxu** [comment]
Cachyos (Arch distro) has a option for open source nvidia drivers if you use GRUB during the installation
